devel/ld64/Portfile

```diff
@@ -253,16 +274,13 @@ if {${subport} eq ${name}} {
         depends_build-append port:cctools
     }
 
-    set all_llvm_variants {llvm39 llvm40 llvm50 llvmdev}
+    set all_llvm_variants {llvm37 llvm39 llvm40 llvm50 llvmdev}
```

Comment:
I don't know how thrilled Jeremy is going to be with the reintroduction of the old llvm 3.7 to Darwin 14 and newer systems. Presumably there is a good reason why the portfile didn't offer that already. The llvm37 variant was deliberately removed in cbfc66ebe8e502f96cc6808c9e1c84f2bd65d6af and was only re-added in e16e82613704613186e523d27b6b2e7bf87484d5 to help bootstrap Mountain Lion and Mavericks.

It sounds like you're reintroducing the llvm37 variant because the libtapi stuff doesn't work with newer llvm versions. But you haven't removed the newer llvm variants. Won't a user who selects one of the newer llvm variants encounter build failures? Wouldn't this include all existing users of newer systems who upgrade, since they will already have one of the newer llvm variants selected?